### ISSUE TITLE
"Something went wrong" when opening an application not installed

### ISSUE THREAD

**Issue by jsilvestre** [cozy/cozy-proxy#97]
The error message lets user thinks something went wrong whereas they are just trying to access an application that is not installed.

It should show the user a proper message, with a link to the marketplace.

**Comment by aenario** [comment]
# Idea : 

Home may be able to handle it better, using the "goto" intent, if the app doesnt exist we display the market / the installation screen for this app.

**Comment by jsilvestre** [comment]
Yeah actually I thought the Home couldn't see the error, but it can since it does the request. +1 @aenario
